scripts/collect-stats.ts

```diff
@@ -75,6 +92,19 @@ class StatsCollector {
         if (!reviewedPRs[login].has(pull_request_url)) {
           contributors[login].reviews[repo.name] =
             (contributors[login].reviews[repo.name] || 0) + 1;
+
+          if (!contributors[login].reviews_by_category[repo.name]) {
+            contributors[login].reviews_by_category[repo.name] = {};
+          }
+
+          for (const key of Object.keys(customCategories)) {
+            for (const glob of customCategories[key]) {
+              if (minimatch(path, glob)) {
+                contributors[login].reviews_by_category[repo.name][key] =
+                  (contributors[login].reviews_by_category[repo.name][key] || 0) + 1;
+              }
+            }
```

Comment:
If you agree with breaking up categories by repo, we’d need to tweak this too, something like:
```suggestion
            for (const repoName in customCategories[key]) {
              if (repoName !== repo.name) continue;
              for (const glob of customCategories[key][repoName]) {
                if (minimatch(path, glob)) {
                  contributors[login].reviews_by_category[repo.name][key] =
                    (contributors[login].reviews_by_category[repo.name][key] || 0) + 1;
                }
              }
            }
```